src/anomalib/__init__.py

```diff
@@ -22,3 +22,4 @@ class TaskType(str, Enum):
     CLASSIFICATION = "classification"
     DETECTION = "detection"
     SEGMENTATION = "segmentation"
+    LANGUAGE = "language"
```

Comment:
This could also be `VISUAL_PROMPTING` task. Have a look at this
https://paperswithcode.com/task/visual-prompting

@ashwinvaidya17 @djdameln

Reply:
Yeah makes sense.